dev/generator/main_test.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	responseToDecode = []string{
+		"attributes.uiStateJSON",
+		"attributes.visState",
+		"attributes.optionsJSON",
+		"attributes.panelsJSON",
+		"attributes.kibanaSavedObjectMeta.searchSourceJSON",
+	}
+)
+
+// DecodeExported decodes an exported dashboard
+//func EncodeKibanaAssets(result MapStr) MapStr {
+
+func TestFoo(t *testing.T) { // Read file from json
```

Comment:
If I'm reading this correctly, the test ensures no errors occur when a) reading the test file b) calling `encodedSavedObject`.

If it doesn't already, it would be useful to test if the output of the function is what's expected; not only that it doesn't fail.

Reply:
Agree. My plan is to have a more static test file and check the generated output to compare it. At the moment I link to an existing file which will probably not be there anymore. Will merge as is and follow up.